.github/workflows/release.yml

```diff
@@ -92,44 +92,102 @@ jobs:
-      
-        docker push valory/open-autonomy-tendermint:$TAG
-        docker push valory/open-autonomy-tendermint:latest
-      
-        docker push valory/open-autonomy-hardhat:$TAG
-        docker push valory/open-autonomy-hardhat:latest
-      
-        docker push valory/open-autonomy-user:$TAG
-        docker push valory/open-autonomy-user:latest
+        docker build -t valory/open-autonomy-docs:$TAG -f deployments/Dockerfiles/documentation/Dockerfile . --push
+        docker build -t valory/open-autonomy-docs:latest -f deployments/Dockerfiles/documentation/Dockerfile . --push
+
+  publish-deploy-images:
+    name: Publish Deploy Images
+    runs-on: ubuntu-latest
+    needs:
+      - publish-autonomy-packages
+    steps:
+      - uses: actions/checkout@v2
+      - name: Docker login
+        env:
+          DOCKER_USER: ${{secrets.DOCKER_USER}}
+          DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+        run: |
+          docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+      - name: Set up support for multi platform build
+        run: |
+          docker run --privileged --rm tonistiigi/binfmt --install all
+          docker buildx create --use --name multibuild
+          docker buildx inspect --bootstrap
+      - name: Set up tag
+        run: echo export TAG=$(python3 -c "from setup import about; print(about[\"__version__\"])") > env.sh
+      - name: Build and push version tagged images
+        run: |
+          # export `TAG` variable
+          source env.sh
+          docker buildx build --platform linux/amd64,linux/arm64 -t valory/open-autonomy:$TAG deployments/Dockerfiles/autonomy --push
+          docker buildx build --platform linux/amd64,linux/arm64 -t valory/open-autonomy:latest deployments/Dockerfiles/autonomy --push
+
+  publish-user-images:
+    name: Publish User Images
+    runs-on: ubuntu-latest
+    needs:
+      - publish-autonomy-packages
+    steps:
+      - uses: actions/checkout@v2
+      - name: Docker login
+        env:
+          DOCKER_USER: ${{secrets.DOCKER_USER}}
+          DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+        run: |
+          docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+      - name: Set up support for multi platform build
+        run: |
+          docker run --privileged --rm tonistiigi/binfmt --install all
+          docker buildx create --use --name multibuild
+          docker buildx inspect --bootstrap
+      - name: Set up tag
+        run: echo export TAG=$(python3 -c "from setup import about; print(about[\"__version__\"])") > env.sh
+      - name: Build and push version tagged images
+        run: |
+          # export `TAG` variable
+          source env.sh
+          docker buildx build --platform linux/amd64,linux/arm64 -t valory/open-autonomy-user:$TAG deployments/Dockerfiles/autonomy-user --push
+          docker buildx build --platform linux/amd64,linux/arm64 -t valory/open-autonomy-user:latest deployments/Dockerfiles/autonomy-user --push
```

Comment:
where are the non-arm/amd images being built?

Reply:
https://github.com/valory-xyz/open-autonomy/pull/2026/files/02c6f3a3c6ecb54981f2df02f15e7c34b867f26f#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R106-R107